kolibri/core/assets/src/views/CorePage/AppBarPage.vue

```diff
@@ -167,6 +215,30 @@
 
   @import '~kolibri-design-system/lib/styles/definitions';
 
+  .v-leave {
+    opacity: 1;
+  }
+
+  .v-leave-active {
+    transition: opacity 0.5s;
+  }
+
+  .v-leave-to {
+    opacity: 0;
+  }
+
+  .v-enter {
+    opacity: 0;
+  }
+
+  .v-enter-active {
+    transition: opacity 0.5s;
+  }
+
+  .v-enter-to {
+    opacity: 1;
+  }
+
```

Comment:
It looks like the above custom transition animation(from L218 to L241) is causing the glitches on scroll. The obvious work around for this is get rid of the transition animation and keep it simple. @marcellamaki any additional thoughts?

Reply:
that sounds like a great, simple solution @akolson - let's do that!